packages/rocketchat-slashcommands-msg/server.coffee

```diff
@@ -0,0 +1,36 @@
+###
+# Msg is a named function that will replace /msg commands
+###
+
+class Msg
+	constructor: (command, params, item) ->
+		if command isnt 'msg' or not Match.test params, String
+			return
+
+		trimmedParams = params.trim()
+
+		usernameOrig = trimmedParams.slice(0, trimmedParams.indexOf(' '))
+		message = trimmedParams.slice(trimmedParams.indexOf(' ') + 1)
+
+		username = usernameOrig.replace('@', '')
+
+		if username is ''
+			return
+
```

Comment:
I fixed the handling of empty messages & added a user notification if either the target username or message are empty.